### PR TITLE
Sorting of Camel Video languages

### PR DESCRIPTION
**Pull request by iheanyi1**
Hi @zregvart 
As outreachy applicant for content organisation, i discovered that some camel videos were in another languages other than English.  I took my time to watch those videos to confirm their various languages. This lead to my discovery of some broken video links which reported. I also search for them on archive.org as suggested to confirm their non existence. Below are the broken video links:

*   [Apache Camel Essential Components](https://blog.christianposta.com/webinar/camel-essential-components-webinar/) _by Christian Posta_ (January 2013)(No more available)

*   [Leverage Enterprise Integration Patterns with Apache Camel and Twitter](https://mnclip.info/w8WtDnZz8ac-leverage-eips-with-apache-camel-and-twitter.html) _by Bruno Borges_ (JavaOne 2012)(No more available)

[JavaOne 2012 - Leverage Enterprise Integration Patterns with Apache Camel and Twitter](http://java.dzone.com/articles/leverage-enterprise) _by Bruno Borges_ (JavaOne 2012)

I also move 3 links to article section where they rightly belong and one link to presentation section

Having completed the changes,  I propose for the file change. I will be glad to correct what need to be corrected.

### CODE REVIEW

**Comment by iheanyi1** [comment]
Hi @zregvart ,

These links were flagged after your update of why PRs were not merging. Which lead to this my second commit:

<<<<<<< patch-1
*   [Facebook integration Apache Camel](https://www.youtube.com/watch?v=evx5y1BnT8c)(EN) - 15 min screenshot showing how to integrate Camel with Facebook. This screenshot is part of a series, with new videos coming up.
*   [Advanced AEM Search - Consuming External Content and Enriching Content with Apache Camel](https://www.gastongonzalez.com/tech-blog/2016/8/1/advanced-aem-search-consuming-external-content-and-enriching-content-with-apache-camel)(EN) - 45 min video from 2016 by Gaston Gonzales.
*   [How to use Heroku and Apache Camel to create open source integrations with SalesForce](https://www.youtube.com/watch?v=8yHM1J9qHWw)(EN) - 30 min video from 2017.
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=dSROap0arRE&feature=youtu.be)(EN) - 55 min video from JPoint Moscow 2018, where Claus gave a presentation.
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=I5jdLFdWi6w)(EN) - 45 min video from JEEConf Kiev 2018, where Claus gave a presentation.
*   [Spring Tips - Apache Camel](https://spring.io/blog/2018/05/23/spring-tips-apache-camel)(EN) - 1 hour video from May 2018 by [Josh Long](https://twitter.com/starbuxman) from Pivotal, whom take a look at using Apache Camel on Spring Boot.
*   [A Look Back at Enterprise Integration Patterns and Their Use into Today's Serverless Cloud Computing](https://www.youtube.com/watch?v=EbTZ5nSJQwI)(EN) - 1h video from Devoxx Poland 2018 by Bruno Borges with l<span style="color: rgb(20,23,26);">ots of demos on **Apache Camel**<span style="color: rgb(20,23,26);">, Azure Functions, and quick look into Kafka and Azure Event Hubs.
*   [Camel microservices with Spring Boot and Kubernetes](https://vimeo.com/289646198)(EN) - 6o min video from JavaZone (September 2018) where Claus gave a presentation. Source code and slides are [available here](https://github.com/davsclaus/camel-riders-in-the-cloud/tree/javazone).
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=G8h7oCGsTVQ&t=10s)(EN) - 50 min video from Voxxed Days Ticino (October 2018) where Claus gave a presentation. Source code and slides are [available here](https://github.com/davsclaus/camel-riders-in-the-cloud/tree/ticino).
=======
*   [Facebook integration Apache Camel](https://www.youtube.com/watch?v=evx5y1BnT8c) - 15 min screenshot showing how to integrate Camel with Facebook. This screenshot is part of a series, with new videos coming up.
*   [Advanced AEM Search - Consuming External Content and Enriching Content with Apache Camel](https://www.gastongonzalez.com/tech-blog/2016/8/1/advanced-aem-search-consuming-external-content-and-enriching-content-with-apache-camel) - 45 min video from 2016 by Gaston Gonzales.
*   [How to use Heroku and Apache Camel to create open source integrations with SalesForce](https://www.youtube.com/watch?v=8yHM1J9qHWw) - 30 min video from 2017.
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=dSROap0arRE&feature=youtu.be) - 55 min video from JPoint Moscow 2018, where Claus gave a presentation.
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=I5jdLFdWi6w) - 45 min video from JEEConf Kiev 2018, where Claus gave a presentation.
*   [Spring Tips - Apache Camel](https://spring.io/blog/2018/05/23/spring-tips-apache-camel) - 1 hour video from May 2018 by [Josh Long](https://twitter.com/starbuxman) from Pivotal, whom take a look at using Apache Camel on Spring Boot.
*   [A Look Back at Enterprise Integration Patterns and Their Use into Today's Serverless Cloud Computing](https://www.youtube.com/watch?v=EbTZ5nSJQwI) - 1h video from Devoxx Poland 2018 by Bruno Borges with l<span style="color: rgb(20,23,26);">ots of demos on **Apache Camel**<span style="color: rgb(20,23,26);">, Azure Functions, and quick look into Kafka and Azure Event Hubs.
*   [Camel microservices with Spring Boot and Kubernetes](https://vimeo.com/289646198) - 6o min video from JavaZone (September 2018) where Claus gave a presentation. Source code and slides are [available here](https://github.com/davsclaus/camel-riders-in-the-cloud/tree/javazone).
*   [Camel microservices with Spring Boot and Kubernetes](https://www.youtube.com/watch?v=G8h7oCGsTVQ&t=10s) - 50 min video from Voxxed Days Ticino (October 2018) where Claus gave a presentation. Source code and slides are [available here](https://github.com/davsclaus/camel-riders-in-the-cloud/tree/ticino).
*    [Best Practices for Middleware and Integration Architecture Modernization with Apache Camel](https://www.youtube.com/watch?v=d1Hr78a7Lww) - 30 min video about 
Apache Camel v3 focused on Camel K, Camel Quarkus and Camel Kafka Connector.

>>>>>>> master

Could it be the reason for the rebase? if yes, do i go ahead with the rebase?

File: content/community/articles.md [hunk 1]
```diff
@@ -2,52 +2,35 @@
...
-*   [In the brain of James Strachan, riding the Camel](http://skillsmatter.com/podcast/design-architecture/riding-camel/JS-2327) - _by_ _[James Strachan](http://macstrac.blogspot.com/)_ (August 2011)
-*   [CamelOne 2011 conference](http://camelone.org/camelone-2012/) (May 2011)
-*   [What Riding the Camel can make integration easier for you](http://vimeo.com/28760458) _by_ _[Claus Ibsen](http://davsclaus.blogspot.com)_ (September 2011)
-*   [Un plongeon dans les frameworks d'intégration Java: Apache Camel et Spring Integration](https://www.youtube.com/watch?v=gkUzmbwJr8k) _by Guillaume Giamarchi and Grégory Boissinot_ (October 2013)
-*   [JEEConf 2011 - What riding the Camel can do to make integration easier for you](http://jeeconf.com/program/camel/) _by Claus Ibsen_ (May 2011).
-*   [Introduction to messaging with ActiveMQ](http://www.java-tv.com/2009/07/24/introduction-to-messaging-with-apache-activemq/) at [Øredev 2008 conference](https://oredev.org/) _by_ _[Bruce Snyder](http://bsnyderblog.blogspot.com/)_
+### Camel Videos(Lang: EN(GLISH), FRENCH, Norwegian, Russian, Spannish)
```
Comment (zregvart) [inline content/community/articles.md]:
```suggestion
### Camel Videos
```

File: content/community/articles.md [hunk 2]
```diff
@@ -2,52 +2,35 @@
...
+
+*   [Camel Riders in the Cloud](https://developers.redhat.com/video/youtube/zuEYtMvHN6g/)(EN) - A 30 minute video from March 2018 where Claus presents using Camel with microservices and containers as cloud native applications, and the best practices around this.
```
Comment (zregvart) [inline content/community/articles.md]:
```suggestion
*   [Camel Riders in the Cloud](https://developers.redhat.com/video/youtube/zuEYtMvHN6g/) (English) - A 30 minute video from March 2018 where Claus presents using Camel with microservices and containers as cloud native applications, and the best practices around this.
```

**Comment by zregvart** [comment]
Work moved over to #293, comments still stand

File: content/community/articles.md (hunk 1 above)
Comment (iheanyi1) [reply content/community/articles.md]: The videos you mention above are active and on the current list,

**Pull request closed without merging**